contracts/ISTokensDescriptor.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MPL-2.0
+pragma solidity 0.8.4;
+
+import {ISTokensManager} from "@devprotocol/i-s-tokens/contracts/interface/ISTokensManager.sol";
+
+interface ISTokensDescriptor {
+	function getTokenURI(ISTokensManager.StakingPosition memory _position)
+		external
```

Comment:
Use `internal`

Reply:
The size of the contract is so large that it cannot be deployed.
So we decided to separate the STokensDescriptor.
Is this a problem?
@aggre